src/views/Team/Team.js

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect, useState } from 'react';
+import { fetchTeamById } from '../../services/teams';
+import TeamDetails from '../../components/Team/TeamDetails';
+
+export default function Team(props) {
+  const [team, setTeam] = useState([]);
```

Comment:
Would set the initial state to an empty object `{}` as team will end up being an object